sample_interface/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"Go-Learning/sample_interface/abc"
+	"fmt"
+)
+
+func main() {
+
+	var temp abc.Price
+	temp = abc.Beemer{Name: "BMW", Yom: 2020, Base_price: 300000}
```

Comment:
Please correct this, syntax error. 
Doesn't tell the purpose of why we should use interface